Data/README.md

```diff
@@ -0,0 +1,198 @@
+# ADDITIONAL RESOURCES
+
+This document contains additional NLP resources relevant to the challenge. It is divided into three categories: data, software and literature. Note these are merely a fraction of NLP resources available: if you would like to contribute, please either submit a pull request. Note we can only publish non-commerical and open-source tools and libraries and we cannot accept datasets published by state and local entities.
+
+For a comprehensive list of NLP sources, see [here](https://github.com/keon/awesome-nlp).
+
+
+## Data
+
+
+### Policy
+
+* [Regulations.gov](https://www.regulations.gov/)
+
+	- This is a platform that publishes all policy changes and regulations passed by the government in any shape or form. They come from through the Federal Register and are associated with a variety of government agencies. Each regulation has a HTM and JSON file. The HTM file contains the regulation in readable format whereas the JSON file contains metadata about the regulation. Some important information in the JSON files include the subject, topics, link to the PDF file, and posted date. 
```

Comment:
- This is a platform that publishes public policy and rule changes made by the federal government in. They come from through the Federal Register and are associated with a variety of contributing and non-contributing federal agencies. Announcement includes a mix of HTML, Word docs, PDFs, and JSON files. The HTML file contains the text of the announcement whereas the JSON file contains metadata. Some important information in the JSON files include the subject, topics, link to attached documents, and posted date.